package.json

```diff
@@ -56,6 +56,7 @@
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-prettier": "4.0.0",
     "eslint-plugin-tsdoc": "0.2.16",
+    "lighthouse": "9.5.0-dev.20220818",
```

Comment:
The output scripts require features in LH 10.0. This dep is only used for testing so it should be fine to use a preview build.